src/Workleap.DomainEventPropagation.Subscription.PullDelivery.Tests/CloudEventHandlerUnitTests.cs

```diff
@@ -50,6 +49,16 @@ private static CloudEvent GivenCloudEvent()
         return cloudEvent;
     }
 
+    private static ICloudEventHandler GivenCloudEventHandler(DomainEventTypeRegistry? registry = null)
+    {
+        var services = new ServiceCollection();
+        return new CloudEventHandler(
+            services.BuildServiceProvider(),
+            registry ?? new DomainEventTypeRegistry(),
+            Enumerable.Empty<IDomainEventBehavior>(),
+            new NullLogger<ICloudEventHandler>());
+    }
```

Comment:
[Suggestion] You could leverage `GSoft.Extensions.Xunit` ([docs](https://github.com/gsoft-inc/wl-extensions-xunit)) to create a fixture with support for `IServiceCollection` and the `ConfigureServices` method to register what you need rather than manually instantiate all injected properties.

Reply:
Won't fix - this piece of code is updated in another PR